config/database.yml

```diff
@@ -14,4 +14,5 @@ test:
   database: <%= ENV.fetch('DATABASE_NAME', 'timeoverflow_test') %>
 
 production:
-  url: <%= ENV.fetch('DATABASE_URL', 'Set DATABASE_URL environment variable') %>
+  # Set DATABASE_URL environment variable
+  url: <%= ENV.fetch('DATABASE_URL', 'null') %>
```

Comment:
`production` key could be removed and Rails also will read the DATABASE_URL for that environment..

other option could be to apply defaults as in the other environments as the DATABASE_URL will always be merged
```yaml
production:
  <<: *defaults
  database: <%= ENV.fetch('DATABASE_NAME', 'timeoverflow') %>
```